test/Microsoft.AspNet.FileProviders.Combined.Tests/CombinedFileProviderTests.cs

```diff
@@ -0,0 +1,252 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using Microsoft.AspNet.FileProviders.Combined.Tests.TestUtility;
+using Xunit;
+using System.Collections.Generic;
+using Microsoft.Extensions.Primitives;
+
+namespace Microsoft.AspNet.FileProviders.Embedded.Tests
+{
+    public class CombinedFileProviderTests
+    {
+        [Fact]
+        public void GetFileInfo_ReturnsNotFoundFileInfo_IfNoFileProviderSpecified()
+        {
+            // Arrange
+            var provider = new CombinedFileProvider();
```

Comment:
I'm not sure that imposing a minimum number of providers is a good thing (it is not in one of my use case: I want the list to be configurable, and the configuration can be to take only one source).